### PR TITLE
feat: add french translation for descriptions

### PR DESCRIPTION
**Pull request by RisingSunLight42**
## Description 

Since I'm french, I've decided to add french description for all the presences where I am owner or contributor.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by RisingSunLight42** [comment]
@Kozou4ever pourrais-tu check les trad ? ^^

**Comment by Kozou4ever** [comment]
Euh si tu veux

File: websites/V/VRV/dist/metadata.json [hunk 1]
```diff
@@ -13,11 +13,12 @@
 	"service": "VRV",
 	"description": {
 		"en": "VRV is the home of your favorite channels, events, and communities celebrating anime, animation, video games, comics, science fiction, fantasy, and tech.",
+		"fr": "VRV est l'endroit pour vos chaînes, événements and communautés favoris autour des animes, de l'animation, des jeux videos, des comics, de la science fiction, de la fantasie et de la technologie.",
 		"nl": "VRV is de thuisbasis van je favoriete kanalen, evenementen en communities die anime, animatie, videogames, strips, science fiction, fantasy en tech vieren.",
 		"ga_IE": "Is é VRV baile na gcainéal, na n-imeachtaí agus na bpobal is fearr leat ag ceiliúradh anime, beochana, cluichí físeáin, greannáin, ficsean eolaíochta, fantaisíochta agus ardteicneolaíochta."
 	},
 	"url": "vrv.co",
-	"version": "2.0.0",
+	"version": "2.1.0",
```
Comment (DarkVillager) [inline websites/V/VRV/dist/metadata.json:21]:
```suggestion
	"version": "2.0.1",
```

File: websites/S/Sitting on Clouds Radio/dist/metadata.json [hunk 2]
```diff
@@ -12,11 +12,12 @@
...
 	"description": {
-		"en": "Video game music radio from the Sitting on Clouds Soundtrack Collection",
-		"nl": "Video game muziek radio van de Sitting on Clouds Soundtrack Collectie"
+		"en": "Video game music radio from the Sitting on Clouds Soundtrack Collection.",
+		"fr": "Radio de musiques de jeux vidéos de la collection de bandes son de Sitting on Clouds.",
+		"nl": "Video game muziek radio van de Sitting on Clouds Soundtrack Collectie."
 	},
 	"url": "radio.sittingonclouds.net",
-	"version": "1.2.0",
+	"version": "1.3.0",
```
Comment (DarkVillager) [inline websites/S/Sitting on Clouds Radio/dist/metadata.json:20]:
```suggestion
	"version": "1.2.1",
```

File: websites/S/Simplyaweeb/dist/metadata.json [hunk 3]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "Simplyaweeb",
 	"description": {
-		"en": "Weebs only one stop to get updated with the latest anime content, meme and bang on the latest anime music. More features coming soon so stay tuned."
+		"en": "Weebs only one stop to get updated with the latest anime content, meme and bang on the latest anime music. More features coming soon so stay tuned.",
+		"fr": "Un seul arrêt pour être à jour avec les derniers animes, memes et la dernière ost. Plus de fonctionnalités sont à venir, donc restez à l'écoute."
 	},
 	"url": "simplyaweeb.to",
-	"version": "1.0.0",
+	"version": "1.1.0",
```
Comment (DarkVillager) [inline websites/S/Simplyaweeb/dist/metadata.json:13]:
```suggestion
	"version": "1.0.1",
```

File: websites/S/SOAP2DAY/dist/metadata.json [hunk 4]
```diff
@@ -24,7 +25,7 @@
 		"s2dfree.de",
 		"s2dfree.is"
 	],
-	"version": "3.0.0",
+	"version": "3.1.0",
```
Comment (DarkVillager) [inline websites/S/SOAP2DAY/dist/metadata.json:28]:
```suggestion
	"version": "3.0.1",
```

File: websites/M/MangaFreak/dist/metadata.json [hunk 5]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "MangaFreak",
 	"description": {
-		"en": "Read manga and download for free. Mangafreak has the latest chapter of all your favorite manga. Check the latest chapter of popular manga like Naruto, Boruto, One Piece and Bleach. Enjoy reading. Manga Reader Online."
+		"en": "Read manga and download for free. Mangafreak has the latest chapter of all your favorite manga. Check the latest chapter of popular manga like Naruto, Boruto, One Piece and Bleach. Enjoy reading. Manga Reader Online.",
+		"fr": "Lisez des mangas et téléchargez-les gratuitement. Mangafreak a le dernier chapitre de tous vos mangas favoris. Lisez le dernier chapitre de mangas populaires comme Naruto, Boruto, One Piece et Bleach. Profitez de votre lecture. Lecteur de manga en ligne."
 	},
 	"url": "w13.mangafreak.net",
-	"version": "1.0.0",
+	"version": "1.1.0",
```
Comment (DarkVillager) [inline websites/M/MangaFreak/dist/metadata.json:13]:
```suggestion
	"version": "1.0.1",
```

File: websites/M/MangaDex/dist/metadata.json [hunk 6]
```diff
@@ -25,14 +25,15 @@
...
+		"fr": "Lit des mangas en ligne gratuitement sur MangaDex sans pubs, haute qualité d'images et supporte les groupes de scanlation !",
 		"nl": "Lees manga gratis online op MangaDex zonder advertenties, afbeeldingen van hoge kwaliteit en ondersteunende scanlatiegroepen!"
 	},
 	"url": [
 		"mangadex.org",
 		"www.mangadex.org",
 		"mangadex.cc"
 	],
-	"version": "3.0.0",
+	"version": "3.1.0",
```
Comment (DarkVillager) [inline websites/M/MangaDex/dist/metadata.json:36]:
```suggestion
	"version": "3.0.1",
```

File: websites/G/Gmail/dist/metadata.json [hunk 7]
```diff
@@ -21,12 +21,13 @@
...
+		"fr": "Gmail est un service d'email gratuit développé par Google. Les utilisateurs peuvent accéder à Gmail sur le web et utiliser des programmes tiers qui permettent de synchroniser leurs emails grâce à des protocols POP ou IMAP. Gmail a commencé une bêta fermée le premier avril 2004 et a terminé sa phase de test le 7 juillet 2009.",
 		"zh_CN": "Gmail是Google开发的免费电子邮件服务。",
 		"nl": "Gmail is een gratis e-mailservice van Google. Gebruikers hebben toegang tot Gmail via beveiligde webmail of via het POP3- of IMAP4-protocol.",
 		"vi_VN": "Gmail là dịch vụ thư điện tử miễn phí được phát triển bởi Google. Người dùng có thể truy cập Gmail trên web và sử dụng phần mềm bên thứ ba để đồng bộ nội dung thư qua phương thức POP hoặc IMAP. Gmail phát hành bản beta đóng kín vào 1 Tháng 4 2004 và kết thúc quá trình thử nghiệm vào 7 Tháng 7 năm 2009."
 	},
 	"url": "mail.google.com",
-	"version": "2.0.0",
+	"version": "2.1.0",
```
Comment (DarkVillager) [inline websites/G/Gmail/dist/metadata.json:30]:
```suggestion
	"version": "2.0.1",
```

File: websites/A/Artfight/dist/metadata.json [hunk 8]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "Artfight",
 	"description": {
-		"en": "The annual art trading game, starting on July 1st!"
+		"en": "The annual art trading game, starting on July 1st!",
+		"fr": "L'annuel jeu d'échange d'art, commence le premier juillet !"
 	},
 	"url": "artfight.net",
-	"version": "1.0.0",
+	"version": "1.1.0",
```
Comment (DarkVillager) [inline websites/A/Artfight/dist/metadata.json:13]:
```suggestion
	"version": "1.0.1",
```

File: websites/A/Animes Vision/dist/metadata.json [hunk 9]
```diff
@@ -4,20 +4,23 @@
...
 		"en": "Animes Vision is one of the largest Brazilian anime sites.",
+		"fr": "Animes Vision est un des plus grand sites d'animés Brésilien.",
 		"pt_BR": "Animes Vision é um dos maiores sites brasileiros de anime.",
 		"ga_IE": "Tá Animes Vision ar cheann de na suíomhanna anime Brasaíle is mó.",
 		"nl": "Animes Vision is een van de grootste Braziliaanse anime sites.",
 		"vi_VN": "Animes Vision là một trong những trang anime lớn nhất Brazil."
 	},
 	"url": "animes.vision",
-	"version": "2.0.0",
+	"version": "2.1.0",
```
Comment (DarkVillager) [inline websites/A/Animes Vision/dist/metadata.json:23]:
```suggestion
	"version": "2.0.1",
```

File: websites/A/AniList/dist/metadata.json [hunk 10]
```diff
@@ -22,7 +23,7 @@
 		"vi_VN": "AniList: Theo dõi, tìm kiếm, chia sẻ Anime & Manga - Đăng ký ngay hôm nay!"
 	},
 	"url": "anilist.co",
-	"version": "2.0.1",
+	"version": "2.1.0",
```
Comment (DarkVillager) [inline websites/A/AniList/dist/metadata.json:26]:
```suggestion
	"version": "2.0.2",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]
The translator script just bumps it as a patch so let's do the same here

File: websites/A/AniList/dist/metadata.json (hunk 10 above)
Comment (RisingSunLight42) [reply websites/A/AniList/dist/metadata.json]: Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

File: websites/V/VRV/dist/metadata.json (hunk 1 above)
Comment (RisingSunLight42) [reply websites/V/VRV/dist/metadata.json]: Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

File: websites/S/Sitting on Clouds Radio/dist/metadata.json (hunk 2 above)
Comment (RisingSunLight42) [reply websites/S/Sitting on Clouds Radio/dist/metadata.json]: Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

File: websites/S/Simplyaweeb/dist/metadata.json (hunk 3 above)
Comment (RisingSunLight42) [reply websites/S/Simplyaweeb/dist/metadata.json]: Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

File: websites/S/SOAP2DAY/dist/metadata.json (hunk 4 above)
Comment (RisingSunLight42) [reply websites/S/SOAP2DAY/dist/metadata.json]: Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

File: websites/M/MangaFreak/dist/metadata.json (hunk 5 above)
Comment (RisingSunLight42) [reply websites/M/MangaFreak/dist/metadata.json]: Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

File: websites/M/MangaDex/dist/metadata.json (hunk 6 above)
Comment (RisingSunLight42) [reply websites/M/MangaDex/dist/metadata.json]: Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

File: websites/G/Gmail/dist/metadata.json (hunk 7 above)
Comment (RisingSunLight42) [reply websites/G/Gmail/dist/metadata.json]: Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

File: websites/A/Artfight/dist/metadata.json (hunk 8 above)
Comment (RisingSunLight42) [reply websites/A/Artfight/dist/metadata.json]: Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

File: websites/A/Animes Vision/dist/metadata.json (hunk 9 above)
Comment (RisingSunLight42) [reply websites/A/Animes Vision/dist/metadata.json]: I was already a contributor so idk
Resolved in 24ca84dc922fbfccb07bcb5643f0f12a8392f9d8

**Review by EncryptedDev — APPROVED** [review]

**Comment by Bas950** [comment]
May be merged after approval of @Kozou4ever

**Review by Bas950 — APPROVED** [review]

File: websites/A/Artfight/dist/metadata.json [hunk 11]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "Artfight",
 	"description": {
-		"en": "The annual art trading game, starting on July 1st!"
+		"en": "The annual art trading game, starting on July 1st!",
+		"fr": "L'annuel jeu d'échange d'art, commence le premier juillet !"
```
Comment (Kozou4ever) [inline websites/A/Artfight/dist/metadata.json:10]:
```suggestion
		"fr": "Le jeu annuel d'échange artistique, débute le 1ier juillet !"
```

File: websites/G/Gmail/dist/metadata.json [hunk 12]
```diff
@@ -21,12 +21,13 @@
 	"service": "Gmail",
 	"description": {
 		"en": "Gmail is a free email service developed by Google. Users can access Gmail on the web and using third-party programs that synchronize email content through POP or IMAP protocols. Gmail started as a limited beta release on April 1, 2004 and ended its testing phase on July 7, 2009.",
+		"fr": "Gmail est un service d'email gratuit développé par Google. Les utilisateurs peuvent accéder à Gmail sur le web et utiliser des programmes tiers qui permettent de synchroniser leurs emails grâce à des protocols POP ou IMAP. Gmail a commencé une bêta fermée le premier avril 2004 et a terminé sa phase de test le 7 juillet 2009.",
```
Comment (Kozou4ever) [inline websites/G/Gmail/dist/metadata.json:24]:
```suggestion
		"fr": "Gmail est un service d'email gratuit développé par Google. Les utilisateurs peuvent accéder à Gmail sur le web et utiliser des programmes tiers qui permettent de synchroniser leurs emails grâce à des protocoles POP ou IMAP. Gmail a commencé avec une version bêta limitée le 1er avril 2004 et a terminé sa phase de test le 7 juillet 2009.",
```

File: websites/M/MangaDex/dist/metadata.json [hunk 13]
```diff
@@ -25,14 +25,15 @@
 	"service": "MangaDex",
 	"description": {
 		"en": "Read manga online for free at MangaDex with no ads, high quality images and support scanlation groups!",
+		"fr": "Lit des mangas en ligne gratuitement sur MangaDex sans pubs, haute qualité d'images et supporte les groupes de scanlation !",
```
Comment (Kozou4ever) [inline websites/M/MangaDex/dist/metadata.json:28]:
```suggestion
		"fr": "Lisez des mangas en ligne gratuitement sur MangaDex sans pubs, des images de haute qualité et supporte les groupes de scantrad !",
```

File: websites/S/Simplyaweeb/dist/metadata.json [hunk 14]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "Simplyaweeb",
 	"description": {
-		"en": "Weebs only one stop to get updated with the latest anime content, meme and bang on the latest anime music. More features coming soon so stay tuned."
+		"en": "Weebs only one stop to get updated with the latest anime content, meme and bang on the latest anime music. More features coming soon so stay tuned.",
+		"fr": "Un seul arrêt pour être à jour avec les derniers animes, memes et la dernière ost. Plus de fonctionnalités sont à venir, donc restez à l'écoute."
```
Comment (Kozou4ever) [inline websites/S/Simplyaweeb/dist/metadata.json:10]:
```suggestion
		"fr": "Weebs sont à un seul arrêt pour être à jour avec les derniers animes, les memes et se balancer sur les dernières musiques d'animes. Plus de fonctionnalités sont à venir, donc restez à l'écoute."
```

File: websites/S/Sitting on Clouds Radio/dist/metadata.json [hunk 15]
```diff
@@ -12,11 +12,12 @@
 	],
 	"service": "Sitting on Clouds Radio",
 	"description": {
-		"en": "Video game music radio from the Sitting on Clouds Soundtrack Collection",
-		"nl": "Video game muziek radio van de Sitting on Clouds Soundtrack Collectie"
+		"en": "Video game music radio from the Sitting on Clouds Soundtrack Collection.",
+		"fr": "Radio de musiques de jeux vidéos de la collection de bandes son de Sitting on Clouds.",
```
Comment (Kozou4ever) [inline websites/S/Sitting on Clouds Radio/dist/metadata.json:16]:
```suggestion
		"fr": "Radio de musique de jeux vidéo de la collection de bandes son Sitting on Clouds.",
```

File: websites/V/VRV/dist/metadata.json [hunk 16]
```diff
@@ -13,11 +13,12 @@
 	"service": "VRV",
 	"description": {
 		"en": "VRV is the home of your favorite channels, events, and communities celebrating anime, animation, video games, comics, science fiction, fantasy, and tech.",
+		"fr": "VRV est l'endroit pour vos chaînes, événements and communautés favoris autour des animes, de l'animation, des jeux videos, des comics, de la science fiction, de la fantasie et de la technologie.",
```
Comment (Kozou4ever) [inline websites/V/VRV/dist/metadata.json:16]:
```suggestion
		"fr": "VRV est la maison de vos chaînes, événements et communautés préférés célébrant l'anime, l'animation, les jeux vidéo, les bandes dessinées, la science-fiction, la fantaisie et la technologie.",
```

**Review by Kozou4ever — CHANGES_REQUESTED** [review]
Hello, I apologize for the delay
Listed below are some desired adjustments as regards french strings
Please let me know if you have any questions :)

File: websites/V/VRV/dist/metadata.json (hunk 16 above)
Comment (Kozou4ever) [reply websites/V/VRV/dist/metadata.json:16]: A la limite ici on peut mettre `les BD` pour la traduction des comics

```suggestion
		"fr": "VRV est la maison de vos chaînes, événements et communautés préférés célébrant l'anime, l'animation, les jeux vidéo, les BD, la science-fiction, la fantaisie et la technologie.",
```

**Review by Kozou4ever — APPROVED** [review]

**Pull request merged**